tests/preload/CMakeLists.txt

```diff
@@ -38,6 +38,11 @@ add_cstyle(tests-preload-config ${CMAKE_CURRENT_SOURCE_DIR}/config/config.c)
 add_check_whitespace(tests-preload-basic ${CMAKE_CURRENT_SOURCE_DIR}/basic/basic.c)
 add_check_whitespace(tests-preload-config ${CMAKE_CURRENT_SOURCE_DIR}/config/config.c)
 
+add_library(setumask SHARED setumask.c)
+set_target_properties(setumask PROPERTIES
+	INCLUDE_DIRECTORIES ${CMAKE_SOURCE_DIR}/src/
+	POSITION_INDEPENDENT_CODE ON)
```

Comment:
I understand that is needed when adding an OBJECT library, as cmake doesn't know yet where that is going to be used.